### PR TITLE
Added slides for lesson 11

### PR DESCRIPTION
**Pull request by alena456**

### CODE REVIEW

File: src/lessons/11_dom_part2/dom_part2.slides.html [hunk 1]
```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <title>Javascript: Dom manipulations part 2</title>
+  <meta name="date" content="2021-04-26" />
```
Comment (guicheffer) [inline src/lessons/11_dom_part2/dom_part2.slides.html]: can we also update the date in here onto 29th?

File: src/lessons/11_dom_part2/dom_part2.slides.html [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+        <section>
+          <h3>In-class exercise!</h3>
+
+          <p><small>1.5 hours!</small> 🕘</p>
+
+          <ul>
+            <li>
+              Display 10 movie titles according to an array of nominees from this years Oscar!
+            </li>
```
Comment (guicheffer) [inline src/lessons/11_dom_part2/dom_part2.slides.html:71]: Btw, here's what I thought of for this: (just created 😛 )

```javascript
const movies = [
    { title: "Judas and the Black Messiah", stars: 5, url: "#" },
    { title: "Ma Rainey’s Black Bottom", stars: 4, url: "#" },
    { title: "Mank", stars: 4, url: "#" },
    { title: "Minari", stars: 4, url: "#" },
    { title: "Nomadland", stars: 4, url: "#" },
    { title: "Promising Young Woman", stars: 3, url: "#" },
    { title: "Sound of Metal", stars: 3, url: "#" },
    { title: "The Father", stars: 3, url: "#" },
    { title: "The Trial of the Chicago 7", stars: 3, url: "#" },
    { title: "The United States vs. Billie Holiday", stars: 3, url: "#" },
]
```

**Review by guicheffer — APPROVED** [review]

File: src/lessons/11_dom_part2/dom_part2.slides.html (hunk 1 above)
Comment (alena456) [reply src/lessons/11_dom_part2/dom_part2.slides.html]: Oh thanks for the hint, I didn't see this.

File: src/lessons/11_dom_part2/dom_part2.slides.html (hunk 2 above)
Comment (alena456) [reply src/lessons/11_dom_part2/dom_part2.slides.html:71]: ah great, then we will use this :)

File: src/lessons/11_dom_part2/dom_part2.slides.html [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+            <li class="fragment">getElementById()<br>
+              <code class="fragment formattedCode" data-trim>document.getElementById('main')</code>
+            </li>
+            <li class="fragment">querySelector()<br>
+              <code class="fragment formattedCode" data-trim>document.querySelector('li')</code>
+              <code class="fragment formattedCode" data-trim>document.querySelector('.someclass')</code>
+            </li>
+            <li class="fragment">innerHTML<br>
+              <code class="fragment formattedCode" data-trim>document.querySelector('li').innerHTML = 'Email <a
+                  href="mom@gmail.com">Mom</a>.';;</code>
```
Comment (oktupol) [inline src/lessons/11_dom_part2/dom_part2.slides.html:44]:
```suggestion
                  href="mailto:mom@gmail.com">Mom</a>.';;</code>
```

**Review by oktupol — APPROVED** [review]
Looks good to me, apart from one little thing 😄

**Pull request merged**